app/models/profile.rb

```diff
@@ -31,6 +31,9 @@ class Profile < ApplicationRecord
     twitch_url
     mastodon_url
     website_url
+    git_lab_url
```

Comment:
```suggestion
    gitlab_url
```

This will fix the failing specs